odd-platform-api/src/test/resources/scripts/dropAll.sql

```diff
@@ -0,0 +1,53 @@
+drop table if exists flyway_schema_history cascade;
```

Comment:
`DO $$ DECLARE r RECORD; BEGIN FOR r IN (SELECT tablename FROM pg_tables WHERE schemaname = current_schema()) LOOP EXECUTE 'TRUNCATE ' || quote_ident(r.tablename) || ' CASCADE'; END LOOP; END $$;`

 We can use this sql script to TRUNCATE all tables in database schema instead of manual specifying table_names